File: html/old/R2/SACO R2 2010 (Solutions)/palin.py
#!/usr/bin/env python
# Keegan Carruthers-Smith - Palin - SACO R2 2010

word = raw_input('Enter the word you wish to see if it is a Palindrome? ')
if word == word[::-1]:
    print 'Is a Palindrome'
else:
    print 'Is Not a Palindrome'

File: html/old/R2/SACO R2 2010 (Solutions)/Palin.java
import java.util.Scanner;

public class Palin {
    public static void main (String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter a word: ");
        String word = sc.next();

        StringBuffer backwards = new StringBuffer(word);
        backwards.reverse();

        if (word.equals(backwards.toString()))
            System.out.println("Is a Palindrome");
        else
            System.out.println("Is Not a Palindrome");
    }
}
